synapse/state/v2.py

```diff
@@ -288,19 +313,23 @@ def _is_power_event(event):
 
 
 async def _add_event_and_auth_chain_to_graph(
-    graph, room_id, event_id, event_map, state_res_store, auth_diff
-):
+    graph: Dict[str, Set[str]],
```

Comment:
I debated if it made sense to define a `Graph` type, it gets used in a handful of places.